src/main/java/com/microsoft/graph/tasks/IProgressCallback.java

```diff
@@ -1,37 +0,0 @@
-// ------------------------------------------------------------------------------
```

Comment:
keep large file upload

Reply:
These classes are also dropped while they are refactored to work with the request adapter pattern.